Sources/SpeziLLM/LLMSessionProvider.swift

```diff
@@ -45,7 +45,7 @@ public struct _LLMSessionProvider<Schema: LLMSchema>: DynamicProperty {     // s
     }
     
     /// Creates a `Binding` to the ``LLMSession``that one can pass around. Useful for passing the ``LLMSession`` as a `Binding` to the ``LLMChatView``.
-    public var projectedValue: Binding<Schema.Platform.Session> {
+    @MainActor public var projectedValue: Binding<Schema.Platform.Session> {
```

Comment:
Just double checking: This `MainActor` annotation shouldn't lead to any issues as the `Session` `Binding` is (currently) only used within SwiftUI `View`s which are (since the last major SwiftUI release) fully on the `MainActor` (not just the `View` `body`)

Reply:
I think so too. I added the @MainActor annotation mainly to avoid the warning, but it's actually ok here since this is supposed to be used within a `View` context.

Reply:
Yes exactly, the `_LLMSessionProvider` is for now only exposed on the `View`, therefore no additional constraints added on the `Binding`.

Reply:
Are you suggesting to remove it? Xcode complains and there is no harm enforcing this.

Reply:
No, not at all, '@MainActor' is the correct isolation mechansim on the 'Binding'.